test/e2e/kubevirt.go

```diff
@@ -285,22 +287,54 @@ var _ = Describe("Kubevirt Virtual Machines", func() {
 			}
 		}
 
-		checkEastWestIperfTraffic = func(vmi *kubevirtv1.VirtualMachineInstance, podIPsByName map[string][]string, stage string) {
+		startEastWestIperfTraffic = func(vmi *kubevirtv1.VirtualMachineInstance, podIPsByName map[string][]string, stage string) error {
 			GinkgoHelper()
 			Expect(podIPsByName).ToNot(BeEmpty())
 			polling := 15 * time.Second
-			timeout := time.Minute
 			for podName, podIPs := range podIPsByName {
 				for _, podIP := range podIPs {
-					output := ""
-					Eventually(func() error {
-						var err error
-						output, err = kubevirt.RunCommand(vmi, fmt.Sprintf("iperf3 -t 1 -c %s", podIP), polling)
-						return err
+					port := iperfIPv4Port
+					if utilnet.IsIPv6String(podIP) {
+						port = iperfIPv6Port
+					}
+					output, err := kubevirt.RunCommand(vmi, fmt.Sprintf("iperf3 -t 0 -c %[2]s -p %[3]d --logfile /tmp/%[1]s_%[2]s_%[3]d_iperf3.log &", podName, podIP, port), polling)
+					if err != nil {
+						return fmt.Errorf("%s: %w", output, err)
+					}
+				}
+			}
+			return nil
+		}
+
+		checkEastWestIperfTraffic = func(vmi *kubevirtv1.VirtualMachineInstance, podIPsByName map[string][]string, stage string) {
+			GinkgoHelper()
+			Expect(podIPsByName).ToNot(BeEmpty())
+			for podName, podIPs := range podIPsByName {
+				for _, podIP := range podIPs {
+					port := iperfIPv4Port
+					if utilnet.IsIPv6String(podIP) {
+						port = iperfIPv6Port
+					}
+					// Check the last line eventually show traffic flowing
+					Eventually(func() (string, error) {
+						iperfLog, err := kubevirt.RunCommand(vmi, fmt.Sprintf("cat /tmp/%s_%s_%d_iperf3.log", podName, podIP, port), 2*time.Second)
```

Comment:
Also please consider setting the console [verbose](https://github.com/ovn-kubernetes/ovn-kubernetes/blob/master/test/e2e/kubevirt/console.go#L191) to OFF as it will produce a lot less unformatted outputs

Reply:
> Doing this writes to the screen the log output every time, which can be a lot .. suggestion: maybe use `tail -n 1` instead of cat? and then only if the eventually fails then print the entire log once (for easier debugging)

I like to see that but only on failure, I will play with knobs to see if we can have that, but I prefer not to use tail and make the command return the full iperf logs so we massage the message with golang, it may also help check amount of disruption.

Reply:
you can add the actual log on the Fail descriptio, something like
```
stage+": failed checking iperf3 traffic at "+iperfLogFile + fmt.Sprintf(": %s", printLog(iperfLogFile))
```

Reply:
There is one thing though, I am also using expecter.VerboseWriter(GinkgoWriter) with means that if you don't pass "-v" option to run you will not see console logs and will only pop up if failure, isn't that enough ?.

I mean since you are running with `-v` you see all the console logs.

Reply:
I think it's good enough, but better see, is it pushed so I can try to see the output?

Reply:
I mean that this is how it works right now, no need to push changes.